components/script/dom/document.rs

```diff
@@ -243,6 +244,26 @@ pub struct Document {
     referrer: Option<String>,
     /// https://html.spec.whatwg.org/multipage/#target-element
     target_element: MutNullableHeap<JS<Element>>,
+    /// https://w3c.github.io/uievents/#event-type-dblclick
+    #[ignore_heap_size_of = ""]
+    last_click_info: DOMRefCell<ClickInfo>,
+}
+
+no_jsmanaged_fields!(ClickInfo);
+struct ClickInfo(Option<(Instant, Point2D<i32>)>);
+
+impl ClickInfo {
+    fn take(&self) -> Option<(Instant, Point2D<i32>)> {
+        let ClickInfo(opt) = *self;
+        match opt {
```

Comment:
@KiChjang Do you mean dropping the `Option` and just having a `(Instant, Point2D<i32>)`?